github3/pulls.py

```diff
@@ -795,18 +823,44 @@ def _update_attributes(self, review):
         self.id = review['id']
         self.author_association = review['author_association']
         self.body = review['body']
-        self.body_html = review['body_html']
-        self.body_text = review['body_text']
+        self.body_html = review.get('body_html')
+        self.body_text = review.get('body_text')
```

Comment:
These appear to be optional if you look at the two test reviews I added, unclear as to why these are different to other pull reviews? Also don't appear in the documentation for response output.

Reply:
This is because the Accept header we use by default requests both of these attributes in every response. It's plausible they don't come back on creation so we maybe need a different class here to differentiate between comments that were just created and other comments.

Reply:
When I was testing manually to see what differences there were between the documented output and the returned output I wasn't setting any additional headers, I'll retest, it's probably returned as expected in that case and I can update the example and drop the use of `.get()` here.